core/controllers/blog_dashboard_test.py

```diff
@@ -26,6 +26,36 @@
 import utils
 
 
+class BlogDashboardPageTests(test_utils.GenericTestBase):
+    """Checks the access to the blog dashboard page and its rendering."""
+
+    def test_blog_dashboard_page_access_without_logging_in(self):
+        """Tests access to the Blog Dashboard page."""
+        self.get_html_response('/blog-dashboard', expected_status_int=302)
+
+    def test_blog_dashboard_page_access_without_having_rights(self):
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+        self.login(self.VIEWER_EMAIL)
+        self.get_html_response('/blog-dashboard', expected_status_int=401)
+        self.logout()
+
+    def test_blog_dashboard_page_access_as_blog_admin(self):
+        self.signup(self.BLOG_ADMIN_EMAIL, self.BLOG_ADMIN_USERNAME)
+        self.set_user_role(
+            self.BLOG_ADMIN_USERNAME, feconf.ROLE_ID_BLOG_ADMIN)
+        self.login(self.BLOG_ADMIN_EMAIL)
+        self.get_html_response('/blog-dashboard')
```

Comment:
Shouldn't we add `expected_status_int=200`?

Reply:
It is set to `expected_status_int=200` by default, however added.